src/main/java/twilightforest/item/TFRecipes.java

```diff
@@ -101,8 +119,19 @@ public static void registerRecipes() {
             GameRegistry.addRecipe(
                     new ItemStack(TFBlocks.woodenSlab, 6, i),
                     new Object[] { "###", '#', new ItemStack(TFBlocks.planks, 1, i) });
+            GameRegistry.addRecipe(
+                    new ItemStack(chests[i], 2),
+                    new Object[] { "###", "#0#", "###", '#', new ItemStack(TFBlocks.planks, 1, i), '0',
+                            new ItemStack(Blocks.chest) });
```

Comment:
we dont want vanilla chest or trapped chest recipes in gtnh

Reply:
yea we need to have some gtnh recipes like for the trap doors and the fences before